ingestion/src/metadata/cli/docker.py

```diff
@@ -65,12 +68,28 @@
 )
 
 
+def docker_volume():
+    # create a main directory
+    if not os.path.exists(MAIN_DIR):
+        os.mkdir(MAIN_DIR)
+        db = "db-data"
+        om_server = "om-server"
```

Comment:
Let's only create docker volumes for `db-data`.

Reply:
Done